stories/bar/2_label_value.tsx

```diff
@@ -61,6 +61,7 @@ export const Example = () => {
         fontStyle: 'normal',
         padding: 0,
         fill: color('value color', '#000'),
+        shadowColor: color('shadow color', 'transparent'),
```

Comment:
I think we should enforce the use of a shadow:

```suggestion
        shadowColor: color('shadow color', 'rgba(0,0,0,0.5)'),
```

Reply:
by default in the chart config or just on this one story?

Reply:
Forgotten to report here, just the story.